tsfile/src/main/java/org/apache/iotdb/tsfile/read/reader/chunk/VectorChunkReader.java

```diff
@@ -59,7 +59,7 @@
 
   protected Filter filter;
 
-  private final List<IPageReader> pageReaderList = new LinkedList<>();
+  private final List<IPageReader> pageReaderList = new ArrayList<>();
```

Comment:
In vectorChunkReader Line 268, `return pageReaderList.remove(0).getAllSatisfiedPageData();`

Reply:
Similarly, this list object is also randomly accessed in a loop